plugins/Over9000/__init__.py

```diff
@@ -0,0 +1,36 @@
+﻿import json
+from random import choice
+
+from discord.ext import commands
+
+from Plugin import AutomataPlugin
+
+
+class Over9000(AutomataPlugin):
+    """Kakarot"""
+
+    def __init__(self, manifest, bot):
+        super().__init__(manifest, bot)
+        with open("plugins/Over9000/quotes.json", "r") as f:
```

Comment:
I'm so sorry about keeping doing this, but this should be changed to not rely on the current working directory. You can access an absolute path to this plugin's folder from `self.manifest["path"]`, and then can use os.path.join to add the quotes.json filename on the end.

Reply:
You need not apologize, I appreciate the feedback.

I now know how to access an absolute path for the plugin folder for any future plugins I may have in mind for this bot.

The manifest is passed into the plugin's constructor, where does the manifest come from?

Reply:
Just curious, thought it might be a json file that gets loaded.

Reply:
The manifest is the plugin.json file that you create, however when it gets loaded the path gets added into it for the purposes of plugins referencing their own folder.
The code that handles this is [here](https://github.com/nint8835/jigsaw/blob/master/jigsaw/PluginLoader.py#L66-L70)

Reply:
Thanks!